packages/core/contracts-ovm/insured-bridge/implementation/OVM_BridgeDepositBox.sol

```diff
@@ -183,17 +185,29 @@ contract OVM_BridgeDepositBox is OVM_CrossDomainEnabled, OVM_Testable {
      * @param recipient L1 address that should receive the tokens.
      * @param l2Token L2 token to deposit.
      * @param amount How many L2 tokens should be deposited.
-     * @param maxFeePct Max fraction of the total `amount` that the depositor is willing to pay as a fee. This number is
-     * scaled by 1e18 to represent the percentage of the total transfer, deducted as a fee. Eg 5% would be 0.05e18.
+     * @param slowRelayFeePct Max fraction of `amount` that the depositor is willing to pay as a slow relay fee.
+     * @param instantRelayFeePct Fraction of `amount` that the depositor is willing to pay as a instant relay fee.
+     * @param quoteDeadline Timestamp, after which, this transaction will revert. The depositor will accept L1 LP fees
+     *      for any utilization amount up to this timestamp.
      */
     function deposit(
         address recipient,
         address l2Token,
         uint256 amount,
-        uint256 maxFeePct
+        uint64 slowRelayFeePct,
+        uint64 instantRelayFeePct,
+        uint64 quoteDeadline
     ) public onlyIfDepositsEnabled(l2Token) {
         require(isWhitelistToken(l2Token), "deposit token not whitelisted");
-        require(maxFeePct <= 1e18, "maxFeePct cannot be over 100% (represented as 1e18)");
+        require(
+            slowRelayFeePct + instantRelayFeePct <= 0.5e18,
+            "sum of slow & instant relayer fees can not exceed 50%"
+        );
```

Comment:
I thought we wanted to make sure each individually was < 0.25, so that we could call `max(fee, 0.25)` on each on L1. I'm cool either way, just thought keeping the constraints separate might make the checks a little simpler to reason about. I could see arguments either way, though.

Reply:
yup yup. you right. I will split it into two requirements for each part.